lib/src/screens/dashboard/widgets/address_page.dart

```diff
@@ -119,30 +115,29 @@ class AddressPage extends BasePage {
 
   @override
   Widget? trailing(BuildContext context) {
-    return Material(
-      color: Colors.transparent,
-      child: IconButton(
-        padding: EdgeInsets.zero,
-        constraints: BoxConstraints(),
-        highlightColor: Colors.transparent,
-        splashColor: Colors.transparent,
-        iconSize: 25,
-        onPressed: () {
-          ShareUtil.share(
-            text: addressListViewModel.uri.toString(),
-            context: context,
-          );
-        },
-        icon: Icon(
-          Icons.share,
-          size: 20,
-          color: Theme.of(context)
-              .accentTextTheme!
-              .displayMedium!
-              .backgroundColor!,
-        ),
-      ),
-    );
+    return !addressListViewModel.hasAddressList
+        ? Material(
```

Comment:
@SamsungGalaxyPlayer  is this intended? I can't see anything regarding this in the ticket

this is mainly removing the share icon from the receive screen for both Monero and Haven wallets